src/client/components/DateField.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import DatePicker from "react-datepicker";
+import moment from "moment";
+
+import "react-datepicker/dist/react-datepicker.css";
```

Comment:
Does react-datepicker not manage its own dependency imports? It's odd that we'd have to import its styling separately like this.

Reply:
I'm following their examples, and it's just a blob of data without them, unfortunately. There might be a better way if one understands what they're doing better than I do, but this is what worked.

Reply:
react-datepicker and its dependencies are nuked from orbit after a little fiddling with native date controls. I'm quite satisfied with that as good enough for now.